### DISCUSSION TITLE
Arrow File

### DISCUSSION THREAD

**Discussion by marianogabitto** [GreenleafLab/ArchR#1810] [Questions / Documentation]
Hi ,
     would it be possible to be refer to a document that describe the arrow files in detail ? So far I know that it is an hdf5 file . I would like to know in detail how the reads are stored, etc.

Thanks

**Comment by rcorces** [comment]
SupFig 1a in the original manuscript is the best we have for this. It is on the to-do list to write up a very explicit map of what goes into Arrow files and where but I havent gotten to it yet.
![image](https://user-images.githubusercontent.com/10343046/213185756-5564d44e-0452-4c72-a09e-99784472fa4c.png)
